awx/main/migrations/0012_move_deprecated_stdout.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.7 on 2017-12-12 15:02
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('main', '0011_blank_start_args'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='DeprecatedJobStdout',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('result_stdout_text', models.TextField(blank=True, default=b'', editable=False)),
+                ('job', models.OneToOneField(editable=False, null=True, on_delete=django.db.models.deletion.CASCADE, related_name='deprecated_result_stdout', to='main.UnifiedJob'))
+            ],
+        ),
+        migrations.RunSQL('INSERT INTO main_deprecatedjobstdout(job_id, result_stdout_text) SELECT id, result_stdout_text FROM main_unifiedjob WHERE result_stdout_text IS NOT NULL;'),
```

Comment:
I've got a feeling @matburt ain't gonna like this :)

That said, I really feel like we should avoid doing this in Python - it's just not going to be fast (or memory-performant) enough.  I'm not sure how long it actually would take postgres to do this for really large datasets - I plan to load a GB or so of stdout and attempt this query.